charmcraft.yaml

```diff
@@ -59,6 +59,10 @@ provides:
     interface: tracing
     optional: true
     description: Receive traces from other charms.
+  receive-profiles:
+    interface: profiling
+    optional: true
+    description: Receive profiles from other charms.
```

Comment:
Any additional useful non-trivial information we could share here?

Reply:
let's turn it around. any additional useful non-trivial information you'd like to see here?
I was looking at the other endpoints descriptions and they're also equally plain.
Also they're misleading e.g. tracing: "Send charm traces to a charmed Tempo instance.", doesn't need to be a Tempo instance, it could be another otel collector.